draft-ietf-acme-acme.md

```diff
@@ -1759,13 +1766,16 @@ Revocation requests are different from other ACME request in that they can be
 signed either with an account key pair or the key pair in the certificate.
 Before revoking a certificate, the server MUST verify that the key used to sign
 the request is authorized to revoke the certificate.  The server SHOULD consider
-at least the following keys authorized for a given certificate:
+at least the following accounts authorized for a given certificate:
 
-* the public key in the certificate.
+* the account that issued the certificate.
 
-* an account key that is authorized to act for all of the identifier(s) in the
+* an account that holds authorizations for all of the identifiers in the
   certificate.
 
+The server SHOULD also consider a revocation request valid if it is signed with
```

Comment:
@jsha But how is this related to the certificate key? If I have the account key, I can still always revoke it? If revocation with the certificate key is an optional feature, I see little sense to have it at all, because clients will then rather choose the "safe" (always working) variant and use the account key.